src/Templating/src/Microsoft.DotNet.Web.ProjectTemplates/WebApi-CSharp.csproj.in

```diff
@@ -15,4 +15,8 @@
     <PackageReference Include="Microsoft.AspNetCore.Authentication.AzureADB2C.UI" Version="${MicrosoftAspNetCoreAuthenticationAzureADB2CUIPackageVersion}" Condition="'$(IndividualB2CAuth)' == 'True'" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
Let's condense item groups. You can merge this with the item group above and remove the Condition.

Reply:
Looks like that's true of some of the other projects as well.